### PR TITLE
variable Phiole config location and docker usage

### PR DESCRIPTION
**Pull request by phenomeus**
you can now change the location of pinhole configs at the top

**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [ ] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
*A detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues*

**How does this PR accomplish the above?:**
*A detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix*

**What documentation changes (if any) are needed to support this PR?:**
*A detailed list of any necessary changes*

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

File: padd.sh [hunk 1]
```diff
@@ -118,7 +121,7 @@ pihole_logo_script_retro_3="${green_text}'  ${red_text}'   ${magenta_text}' ${ye
 
 GetFTLData() {
     local ftl_port LINE
-    ftl_port=$(cat /run/pihole-FTL.port 2> /dev/null)
+    ftl_port=$(docker exec pihole cat /run/pihole-FTL.port 2> /dev/null)
```
Comment (PromoFaux) [inline padd.sh:124]: And if the user does not have docker....?

Comment (phenomeus) [reply padd.sh:124]: :/ sorry, that should also be configured above, but I'm too stupid to do that :D

Comment (thomasmerz) [reply padd.sh:124]: @phenomeus Take a sneak preview on https://github.com/yubiuser/pihole_adlist_tool/blob/master/pihole_adlist_tool#L238-L252 to **determine if running in docker** or not.

Then you could do this:
```
if [ "$PIHOLE_DOCKER" = 0 ];
    then
        ftl_port=$(cat /run/pihole-FTL.port 2> /dev/null)
    else
        ftl_port=$(docker exec pihole cat /run/pihole-FTL.port 2> /dev/null)

fi
```

But what is this useful for? I think, it's better and easier to **run `padd.sh` within the docker container** as I mentioned in [PR #155](https://github.com/pi-hole/PADD/pull/155#issuecomment-1045327841).

**Comment by yubiuser** [comment]
I'm going to close this PR. There has been no activity for >1 year now and merge conflicts exist. If you feels this PR should be re-opend, please resolve conflicts first and then reopen.

Thanks for your contribution.

**Pull request closed without merging**